src/method.jl

```diff
@@ -141,7 +141,11 @@ mutable struct PercivalSolver{V} <: AbstractOptimizationSolver
   sub_solver::TronSolver
 end
 
-function PercivalSolver(nlp::AbstractNLPModel{T, V}; subproblem_modifier = identity) where {T, V}
+function PercivalSolver(
+  nlp::AbstractNLPModel{T, V};
+  subproblem_modifier = identity,
+  kwargs...,
```

Comment:
```suggestion
  subsolver_kwargs = Dict(:max_cgiter => nlp.meta.nvar),
```
Like the other calls.